frontend/src/bundles/common/components/range-slider/range-slider.tsx

```diff
@@ -0,0 +1,60 @@
+import 'rc-slider/assets/index.css';
+import '../../../../assets/css/variables/color-variables.css';
+
+import Slider from 'rc-slider';
+import React, { useCallback, useState } from 'react';
+
+import styles from './styles.module.scss';
+
+interface RangeSliderProperties {
+    onChange?: (range: { min: number; max: number }) => void;
+    rangeLimits?: { min: number; max: number };
+}
+
+const RangeSlider: React.FC<RangeSliderProperties> = ({
+    onChange,
+    rangeLimits = { min: -100, max: 1000 },
+}) => {
+    const [range, setRange] = useState([rangeLimits.min, rangeLimits.max]);
+
+    const handleSliderChange = useCallback(
+        (value: number[]): void => {
+            const newRange = value as [number, number];
+            setRange(newRange);
+            if (onChange) {
+                onChange({ min: newRange[0], max: newRange[1] });
+            }
+        },
+        [onChange],
+    );
+
+    const trackStyle = {
+        backgroundColor: 'var(--color-pink-100)',
+    };
+
+    const handleStyle = {
+        borderColor: 'var(--color-pink-100)',
```

Comment:
I would rather create one 'styles' object with trackStyle and handleStyle. But I don't insist, so it's up to you